streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -1285,62 +1257,6 @@ public void shouldNotAllowNullValueJoinerWithKeyOnLeftJoinWithGlobalTableWithNam
         assertThat(exception.getMessage(), equalTo("joiner can't be null"));
     }
 
-    @SuppressWarnings({"unchecked", "deprecation"}) // specifically testing the deprecated variant
-    @Test
-    public void testNumProcesses() {
```

Comment:
I think we should keep this keep, but just update it to remove `through()`.

Reply:
Okay, I have also patched this in the latest commit